### ISSUE TITLE
Problem: Strange behavior of the toolbar

### ISSUE THREAD

**Issue by Angelk90** [cesardeazevedo/react-native-collapsing-toolbar#4]
Problem:
http://i.imgur.com/ZOsElB5.gifv

For the code I was based on the following example:
https://github.com/cesardeazevedo/react-native-collapsing-toolbar/blob/master/example/index.android.js

The problem happens when I click on the image, which does the function:
```
    _onImg(row){
      this.props.navigation.navigate('Img', {
        data: row
      });
    }
```
Then when I go back, the problem happens.

**Comment by jgcmarins** [comment]
@Angelk90 how did you solve?

**Comment by cesardeazevedo** [comment]
Sorry for didn't pay attention for this issue, but this was a problem that remains for a while, and i hadn't be able to take a deep investigation.

i just made some tests on this pull request #8, and i can confirm that it will fix most of those problems.

i will merge it and make a new release today.

**Comment by cesardeazevedo** [comment]
Shipped at `1.0.0-beta.8` looking for feedbacks.
